src/module-elasticsuite-tracker/ViewModel/Notification.php

```diff
@@ -29,35 +27,26 @@
 class Notification implements ArgumentInterface
 {
     /**
-     * @var ElasticsuiteNotification
-     */
-    private $canViewNotificationElasticsuite;
-
-    /**
-     * @var AdminAnalyticsNotification
+     * Prefix for admin analytics cache
+     *
+     * @var string
      */
-    private $canViewNotificationAnalytics;
+    private static $adminAnalyticsCachePrefix = 'admin-usage-notification-popup';
```

Comment:
@PierreGauthier is this variable really used ? PHPMD is complaining on the 2.9.x based PR : https://github.com/Smile-SA/elasticsuite/runs/4805899394?check_suite_focus=true